WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.java

```diff
@@ -701,20 +791,80 @@ public void onFailure(int statusCode) {
     }
 
     /*
+     * post slugs resolution to IDs has completed
+     */
+    @SuppressWarnings("unused")
+    public void onEventMainThread(ReaderEvents.PostSlugsRequestCompleted event) {
```

Comment:
Ah, I had overlooked the fact that this is a single post in the pager activity. In that case, no check should be necessary. Sorry for the false alarm.